core/tests/build_sources/assets/constants.js

```diff
@@ -522,8 +522,6 @@ var constants = {
 
   "ENABLE_PREREQUISITE_SKILLS": false,
 
-  "ENABLE_NEW_STRUCTURE_PLAYERS": false,
```

Comment:
Oh, was this not used anywhere? How was this still false?

Reply:
I think this one was used in the tests.